containers/proxy-tftpd-image/Dockerfile

```diff
@@ -0,0 +1,32 @@
+#!BuildTag: uyuni/proxy-tftpd:latest uyuni/proxy-tftpd:%PKG_VERSION% uyuni/proxy-tftpd:%PKG_VERSION%.%RELEASE% uyuni/proxy-tftpd:beta
+
+FROM registry.suse.com/suse/sle15:15.3
+
+ARG PRODUCT_REPO
+
+# Add distro and product repos
+COPY add_repos.sh /usr/bin
+RUN sh add_repos.sh ${PRODUCT_REPO}
+
+# Build Service required labels
+# labelprefix=org.opensuse.uyuni.proxy-tftpd
+LABEL org.opencontainers.image.title="Proxy tftpd container"
+LABEL org.opencontainers.image.description="Image contains a Proxy component to serve tftp requests"
+LABEL org.opencontainers.image.created="%BUILDTIME%"
+LABEL org.opencontainers.image.vendor="Uyuni project"
+LABEL org.opencontainers.image.url="https://www.uyuni-project.org/"
+LABEL org.opencontainers.image.version="%%PKG_VERSION%%"
+LABEL org.openbuildservice.disturl="%DISTURL%"
+LABEL org.opensuse.reference="registry.opensuse.org/uyuni/proxy-tftpd:%PKG_VERSION%.%RELEASE%"
+# endlabelprefix
+
+# tftp
+EXPOSE 69/udp
+
+VOLUME "/etc/uyuni"
```

Comment:
tftpd container should have access to the same `/srv/tftpboot` volume as http proxy has.

http container will prepare it (cobbler wsgi scripts will) and tftp container then distribute it over tftp to pxe boot clients.